backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -57,35 +54,34 @@ public UserController(ILogger<UserController> logger, IConfiguration configurati
         /// <param name="page"></param>
         /// <param name="quantity"></param>
         /// <param name="sort"></param>
-        /// <returns>Paged object with an array of users.</returns>
-        [HttpGet("/api/admin/users")]
-        public IActionResult GetUsers(int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
+        /// /// <returns>Paged object with an array of users.</returns>
+        [HttpGet ("/api/admin/users")]
+        public IActionResult GetUsers (int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
```

Comment:
What code editor are you using? It seems your code is getting "auto-formatted" the wrong way. I see a lot of unnecessary spaces like the above `GetUsers SPACE (...)`

I see it everywhere

Reply:
done